### PR TITLE
feat(SortBy): update sort by composition in large mode

### PR DESCRIPTION
**Pull request by nicomaligne**
**What is the problem this PR is trying to solve?**
The sortBy UI in the compound list is different from the one in the legacy list.
~~Waiting for https://github.com/Talend/ui/pull/2381~~
**What is the chosen solution to this problem?**
Update it ;) 
**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/List/ListComposition/SortBy/SortBy.component.js [hunk 1]
```diff
@@ -34,45 +68,61 @@ function SortBy(props) {
...
-					onClick={onOrderChange}
-					aria-label={t('LIST_CHANGE_SORT_BY_ORDER', {
-						defaultValue: 'Change sort order. Current order: {{sortOrder}}.',
-						sortOrder: orderLabel,
-					})}
-				>
-					{orderLabel}
-				</NavItem>
+				{options.length === 1 ? (
+					<li className="navbar-text">{options[0].name}</li>
```
Comment (sgangnant) [inline packages/components/src/List/ListComposition/SortBy/SortBy.component.js]: Is this acceptable to have an `<li />` tag being a child of a `<Nav />` component ? Even though `<Nav />` might wrap or not its children with `<ul/>`

Comment (nicomaligne) [reply packages/components/src/List/ListComposition/SortBy/SortBy.component.js]: To be honest I copy paste the sort by code from legacy list

Comment (nicomaligne) [reply packages/components/src/List/ListComposition/SortBy/SortBy.component.js]: If I run an audit on the page with lighhouse I don't have any warning about this.

Comment (nicomaligne) [reply packages/components/src/List/ListComposition/SortBy/SortBy.component.js]: I made some tweak, w3c validator wasn't happy with a part.

**Pull request merged**